validation/pipeline/qase/qase.go

```diff
@@ -0,0 +1,174 @@
+package qase
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/antihax/optional"
+	"github.com/sirupsen/logrus"
+	qase "go.qase.io/client"
+)
+
+type TestCaseSteps struct {
+	Action         string `json:"action,omitempty"`
+	ExpectedResult string `json:"expected_result,omitempty"`
+	InputData      string `json:"input_data,omitempty"`
+	Position       int32  `json:"position,omitempty"`
+}
+
+type TestCase struct {
+	Description string          `json:"description,omitempty"`
+	Title       string          `json:"title"`
+	SuiteId     int64           `json:"suite_id,omitempty"`
+	Automation  int32           `json:"automation,omitempty"`
+	Steps       []TestCaseSteps `json:"steps,omitempty"`
+}
+
+var (
+	qaseToken = os.Getenv(QaseTokenEnvVar)
+	Client    *qase.APIClient
+)
+
+// SetupQaseClient creates a new Qase client from the api token environment variable QASE_AUTOMATION_TOKEN
```

Comment:
instead of being read in as an environment variable, can this be part of the RANCHER_CATTLE_CONFIG config.yaml and read in that way? Like we do for rancher objects and such.

Reply:
I like using an environment variable here instead of a config as this is sensitive data and should just be part of the running "environment," so to speak.